src/osd/OSD.cc

```diff
@@ -1782,20 +1782,20 @@ class OSDSocketHook : public AdminSocketHook {
   OSD *osd;
 public:
   explicit OSDSocketHook(OSD *o) : osd(o) {}
-  bool call(std::string command, cmdmap_t& cmdmap, std::string format,
+  bool call(std::string admin_command, cmdmap_t& cmdmap, std::string format,
```

Comment:
Consider the console commands, around line 5509 or so. It was not obvious to me which of these were "admin" vs. "console" commands at first, and having the above change in would have saved me quite a bit of time. That said, the reason the change is in a separate commit is because it's indeed optional; if there are a lot of strong feelings against it, we don't need to include it, but I know I would have found it helpful.